app/vmselect/promql/eval.go

```diff
@@ -118,7 +118,7 @@ type EvalConfig struct {
 	Step  int64
 
 	// MaxSeries is the maximum number of time series, which can be scanned by the query.
-	// Zero means 'no limit'
+	// Zero means 'no limit' on vmselect side and let vmstorage to calculate.
 	MaxSeries int
```

Comment:
no need in this change, MaxSeries can account for various limits, not only maxUniqueTimeSeries